c3po/core.py

```diff
@@ -52,8 +54,16 @@ def get_feed():
     response = REQ_SESSION.get(request_url, params=PAYLOAD)
     if response.status_code == 400:
         refresh_short_token()
-    print(response.json())
-    return response.json()
+    dict = response.json()
+
+    while ('paging' in dict):
+        x = dict['paging']['next']
+        request_url = x + LTTK_GROUP_ID + '/feed'
+        response = REQ_SESSION.get(request_url, params=PAYLOAD)
+        print(dict)
```

Comment:
From now on, use print statements only while debugging. Remove all print statements before pushing the code.